src/styles/_catalog.scss

```diff
@@ -1,22 +1,36 @@
+$card-width: 200px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
 
   max-width: 944px;
-  margin: 0 auto;
```

Comment:
why did you delete this?)

Reply:
I wanted to center the blocks in other media and accidentally deleted it, I'll put it back now xD